### PR TITLE
implementation of Attendance Page

### PR DESCRIPTION
**Pull request by pacifiquemboni**
### What does this PR do?
[x] Allowing trainee to see attendance
[x] Fixing ratings and performance

### How should this be manually tested?
To test this you need to clone this project and switch to `attendance-page` branch, install all dependencies `npm install` then start the project `npm start`.
On your app first Login before taking atendance. use web and take attendance for that trainee and check if you are able to see recorded attendance

To test ratings and performance, you have to make sure trainee is rated by whether ttl or coordinator from web app then login on the mobile to see ratings and performance

### CODE REVIEW

File: AppManager.tsx
Comment (fredshema) [inline AppManager.tsx:1]: Remove this file it is no longer needed

File: app/dashboard/trainee/attendance.tsx [hunk 1]
```diff
@@ -1,9 +1,191 @@
...
+      },
+    },
+  });
+
+  useEffect(() => {
+    if (data && data.getTraineeAttendance && data.getTraineeAttendance.phases.length > 0) {
+      setSelectedWeek(data.getTraineeAttendance.phases[0].weeks[0].week || null);
+    }
+    if (error) {
+      console.log("Error fetching attendance data:", error);
```
Comment (fredshema) [inline app/dashboard/trainee/attendance.tsx]: Add a toast for a user if fetching fails

**Comment by Mbabazi12** [comment]
@pacifiquemboni why is the table still white in dark mode and even the filter bar

**Review by aimeudoxie — COMMENTED** [review]
Use appropriate branch naming if it's a Feature , Chore or a Fix

**Review by Mbabazi12 — COMMENTED** [review]
@pacifiquemboni  why is the attendance not being fetched do we still have issues on the backend? , Also the filter of  weeks is not working.

**Pull request closed without merging**